client/client.go

```diff
@@ -33,14 +33,14 @@ const (
 )
 
 func init() {
-	data.SetDefaultExpiryTimes(notary.NotaryDefaultExpiries)
+	data.SetDefaultExpiryTimes(data.NotaryDefaultExpiries)
```

Comment:
Let me know if you see any issues with this approach, or if you think NotaryDefaultExpiries belongs someplace else.